sdk/src/Core/Amazon.Runtime/Internal/Util/Logger.cs

```diff
@@ -60,14 +58,54 @@ private void ConfigsChanged(object sender, PropertyChangedEventArgs e)
         private void ConfigureLoggers()
         {
             LoggingOptions logging = AWSConfigs.LoggingConfig.LogTo;
+            bool found = true;
```

Comment:
Shouldn't this be `bool found = false`? There is no way for found to be false currently.